.github/workflows/repair-pr-453-ownership-safety.yml

```
@@ -0,0 +1,311 @@
+          def replace_once(text: str, old: str, new: str, label: str) -> str:
+              count = text.count(old)
+              if count != 1:
+                  raise SystemExit(f"expected exactly one {label} match, found {count}")
+              return text.replace(old, new, 1)
+
+          main_path = Path('src/main/kotlin/html4tree/main.kt')
+          main = main_path.read_text(encoding='utf-8')
+          main = replace_once(
+              main,
+              '''    override fun run() {
+        go(topDir, maxLevel, forceOverwrite, cleanup, dryRun)
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)